.vimrc

```diff
@@ -0,0 +1,15 @@
+set tabstop=4
+set softtabstop=4
+set shiftwidth=4
+set noexpandtab
```

Comment:
```suggestion
```

I recommend editorconfig-vim, which will do this automatically now that we have a .editorconfig.
It will also use the correct 2-space indent for .yml and .clang-format files.

Reply:
Thanks, I'll look into it.